src/CribblyBackend.DataAccess/Games/Repositories/GameRepository.cs

```diff
@@ -99,8 +99,6 @@ public async Task Create(Game game)
         }
         public async Task<Game> Update(Game game)
         {
-            using var scope = new TransactionScope();
-
```

Comment:
Why did you opt to remove the transaction? Since updating a game involves modifying two tables, I figured we'd want it to be seen as an atomic operation from the perspective of other DB accessors

Reply:
It kept erroring our for me, saying the transaction had already completed or something like that. I can put it back in, I just wasn't quite sure how to get it to work and everything worked without it, lol